rbc/tests/test_omnisci.py

```diff
@@ -77,6 +77,20 @@ def farhenheit2celcius(f):
     assert_equal(farhenheit2celcius(40).execute(), np.float32(40 / 9))
 
 
+@pytest.mark.parametrize('dtype', ('float32', 'float64', 'int32', 'int64'))
+def test_direct_call_array(omnisci, dtype):
+    from rbc.omnisci_backend import mean
+    omnisci.unregister()
+
+    @omnisci('T(T[])', T=['float32', 'float64', 'int64', 'int32'], devices=['cpu'])
+    def farhenheit2celcius(f):
```

Comment:
To resolve CI failures, try also renaming `farhenheit2celcius`, say, to `farhenheit2celcius_arr`.